### ISSUE TITLE
Is anyone using this API on production in 2024?

### ISSUE THREAD

**Issue by codewithvk** [splitbee/notion-api-worker#84]

**Comment by willhoney7** [comment]
Yes, we are. Works great although it is fairly limited.

**Comment by janniks** [comment]
Yes, a bunch of folks still are. The public API really hasn‘t come far enough to replace this yet. So most hope Notion won‘t change or cutoff access anytime soon.

**Comment by nayakayp** [comment]
Yes, it's me. Working great so far
